### ISSUE TITLE
Highlight more than one author?

### ISSUE THREAD

**Issue by xuhdev** [jspitz/biblatex-publist#8]
I can think of  two scenarios where people want to highlight more than one author.

- A CV for a research team (highlight team members in case of collaboration).
- Equal contribution (people often append `*` to there last names to indicate that which is considered different names in this package).

**Comment by xuhdev** [comment]
One way to address this issue may be to allow regex in plauthorname

**Comment by jspitz** [comment]
This would require a major rewrite of the plauthor code (using a list), for which I do not have the time currently. Sorry.

**Comment by xuhdev** [comment]
@jspitz Have you thought about just using regex? That can in practice avoid using list.

**Comment by jspitz** [comment]
I can't see how that should work, since we need to parse the name list one by one anyway.

**Comment by xuhdev** [comment]
@jspitz It would not be a name list: it's just a regex. For example, if someone wants a list, they can just do `{Adam,Billy,Charles}`.

**Comment by jspitz** [comment]
You misunderstood me. We have to deal with lists _internally_. A regexp would not make things easier here.

**Comment by xu-cheng** [comment]
FYI, it could be done using biber annotation. So you will declare the highlight author inside bib file as:
```
author+an = {1=highlight},
```

Then you can tweak the style using something like the following:
```
\renewcommand*{\mkbibnamegiven}[1]{\ifitemannotation{highlight}{\textbf{#1}}{#1}}
\renewcommand*{\mkbibnamefamily}[1]{\ifitemannotation{highlight}{\textbf{#1}}{#1}}
```

Though, I don't know how to make this method compatible with `biblatex-publist`.

**Comment by IsaacWang88** [comment]
If some one named First Name Lastname may publish paper using different formats, like F Lastname, F N Lastname, or First Name Lastname.
Are there any simple way to highlight this author by using wildcard in `\plauthorname` macro? The code may look like this:`\plauthorname[F*][]{Lastname}`.

**Comment by jspitz** [comment]
> If some one named First Name Lastname may publish paper using different formats, like F Lastname, F N Lastname, or First Name Lastname.
> Are there any simple way to highlight this author by using wildcard in `\plauthorname` macro? The code may look like this:`\plauthorname[F*][]{Lastname}`.

Yes. See option `plauthorfirstinit`.

**Comment by IsaacWang88** [comment]
> > If some one named First Name Lastname may publish paper using different formats, like F Lastname, F N Lastname, or First Name Lastname.
> > Are there any simple way to highlight this author by using wildcard in `\plauthorname` macro? The code may look like this:`\plauthorname[F*][]{Lastname}`.
> 
> Yes. See option `plauthorfirstinit`.

Yes! After I updated the package, option `plauthorfirstinit` works great! Thank you! It helps us build a perfect CV.

**Comment by jspitz** [comment]
So this is implemented in version 1.14 (which will be pushed to CTAN today)

**Comment by xuhdev** [comment]
Awesome! Thanks for your work!
